### PR TITLE
fix(indexer): Update prisma schema to latest

### PR DESCRIPTION
**Pull request by roninjin10**
- prisma schema was very out of data
- fix via starting the db and running ui/prisma.sh script

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6855** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6855" target="_blank">Graphite</a> 
    * **PR #6857** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6857" target="_blank">Graphite</a> 
      * **PR #6858** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6858" target="_blank">Graphite</a> 
        * **PR #6859** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6859" target="_blank">Graphite</a> 
          * **PR #6860** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6860" target="_blank">Graphite</a> 
            * **PR #6861** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6861" target="_blank">Graphite</a>  👈
              * **PR #6862** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6862" target="_blank">Graphite</a> 
                * **PR #6874** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6874" target="_blank">Graphite</a> 
      * **PR #6856** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6856" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6861?utm_source=stack-comment).

File: indexer/ui/schema.prisma [hunk 1]
```diff
@@ -4,106 +4,203 @@ generator client {
...
-  deposits      deposits[]
-  state_batches state_batches[]
+/// This table contains check constraints and requires additional setup for migrations. Visit https://pris.ly/d/check-constraints for more info.
+model l1_block_headers {
+  hash               String               @id @db.VarChar
+  parent_hash        String               @db.VarChar
+  number             Decimal              @db.Decimal
+  timestamp          Int
+  rlp_bytes          String               @db.VarChar
+  l1_contract_events l1_contract_events[]
```
Comment (hamdiallam) [inline indexer/ui/schema.prisma:37]: Is this a way for prisma to join with the `l1_contract_events` table to make viewing things easy?

**Review by hamdiallam — APPROVED** [review]

File: indexer/ui/schema.prisma (hunk 1 above)
Comment (roninjin10) [reply indexer/ui/schema.prisma:37]: there is a way to do most things in prisma but I'm not a power user

Comment (hamdiallam) [reply indexer/ui/schema.prisma:37]: i meant the comment more in terms of `l1_contract_events` is not a column on `l1_block_headers`. Would this cause an error or does prisma do a join?

File: indexer/ui/schema.prisma [hunk 2]
```diff
@@ -4,106 +4,203 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url      = "postgresql://db_username:db_password@localhost:5434/db_name"
```
Comment (roninjin10) [inline indexer/ui/schema.prisma]:
```suggestion
  url      = env("DATABASE_URL")
```
should keep using DATABASE_URL env var. Might want to update script to be smart enough to do this

**Pull request merged**